lib/azure/armrest/armrest_service.rb

```diff
@@ -166,7 +166,8 @@ def tenants
       # such as create or delete.
       #
       def poll(response)
-        url = response.respond_to?(:azure_asyncoperation) ? response.azure_asyncoperation : response.to_s
+        return 'Succeeded' if [200, 201].include?(response.response_code)
```

Comment:
For my curiosity, what is the response_code if the aync operation is not finished yet?
Also, for the case the response contains `azure_asyncoperation` or `location`, what is the return code when it is still ongoing or finished?